packages/api/src/pubsub.ts

```diff
@@ -14,7 +14,9 @@ export class PubSub {
 		pubsub.subscribe(route, callback)
 	}
 
+	// TODO: more type inconsistencies coming from your lib glenn, fix it >.<
```

Comment:
What is inconsistent?

Reply:
![image](https://github.com/user-attachments/assets/3c9460e0-d11b-4875-a805-91e44adcb3cc)

Why not just `any`

Reply:
That's weird, I was expecting it to be any yes.

Reply:
easy fix though

Reply:
https://github.com/VandeurenGlenn/little-pubsub/commit/5067e4e022c44f05b57fd1c4474316692edc34ae fixes this